build-win32.txt

```diff
@@ -0,0 +1,19 @@
+[binaries]
+c = 'i686-w64-mingw32-gcc'
+cpp = 'i686-w64-mingw32-g++'
+ar = 'i686-w64-mingw32-ar'
+strip = 'i686-w64-mingw32-strip'
+strip = 'i686-w64-mingw32-cmake'
```

Comment:
Shouldn't this be `cmake = 'i686-w64-mingw32-cmake'`?

Reply:
Yes, I have fixed this now. Good spot.